packages/core/bootstrap/src/lib/ws/actions.ts

```diff
@@ -0,0 +1,42 @@
+import { AdapterRequest, WSSubscriptionHandler } from '@chainlink/types'
+import { createAction } from '@reduxjs/toolkit'
+import { asAction } from '../store'
+import { WSConfig, WSConnectionInfo } from './types'
+
+/** CONNECTIONS */
+export interface WSConfigPayload {
+  config: WSConfig
+  wsHandler: WSSubscriptionHandler
```

Comment:
Do we use this?

A handler is an object/function, meaning not serializable, and we should not put any non-serializable data in an event.

```suggestion
```

Reply:
#469